[Bug][Sort] NPE when run MySqlLoadSqlParseTest

### What happened

When running `MySqlLoadSqlParseTest`, there was an error in mysql cdc. 
```
Caused by: java.lang.NullPointerException
	at io.debezium.connector.mysql.MySqlValueConverters.schemaBuilder(MySqlValueConverters.java:167) ~[debezium-connector-mysql-1.5.4.Final.jar:1.5.4.Final]
	at io.debezium.relational.TableSchemaBuilder.addField(TableSchemaBuilder.java:360) ~[debezium-core-1.5.4.Final.jar:1.5.4.Final]
	at io.debezium.relational.TableSchemaBuilder.lambda$create$2(TableSchemaBuilder.java:119) ~[debezium-core-1.5.4.Final.jar:1.5.4.Final]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) ~[?:1.8.0_302]
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175) ~[?:1.8.0_302]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1384) ~[?:1.8.0_302]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482) ~[?:1.8.0_302]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472) ~[?:1.8.0_302]
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150) ~[?:1.8.0_302]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173) ~[?:1.8.0_302]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_302]
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:485) ~[?:1.8.0_302]
	at io.debezium.relational.TableSchemaBuilder.create(TableSchemaBuilder.java:117) ~[debezium-core-1.5.4.Final.jar:1.5.4.Final]
	at io.debezium.relational.RelationalDatabaseSchema.buildAndRegisterSchema(RelationalDatabaseSchema.java:130) ~[debezium-core-1.5.4.Final.jar:1.5.4.Final]
	at io.debezium.relational.HistorizedRelationalDatabaseSchema.recover(HistorizedRelationalDatabaseSchema.java:52) ~[debezium-core-1.5.4.Final.jar:1.5.4.Final]
```

From the source code,  mysql depends on debezium version 1.5.4-Final, while oracle and mongodb depend on 1.6.4-Final. When running `MySqlLoadSqlParseTest`,  debezium-1.6.4-Final is used first, causing a NullPointerException error. 

The temporary solution is to comment out the dependencies of oracle and mongodb in the pom.xml.

```xml
<!--        <dependency>-->
<!--            <groupId>org.apache.inlong</groupId>-->
<!--            <artifactId>sort-connector-mongodb-cdc</artifactId>-->
<!--            <version>${project.version}</version>-->
<!--            <scope>test</scope>-->
<!--        </dependency>-->
<!--        <dependency>-->
<!--            <groupId>org.apache.inlong</groupId>-->
<!--            <artifactId>sort-connector-oracle-cdc</artifactId>-->
<!--            <version>${project.version}</version>-->
<!--            <scope>test</scope>-->
<!--        </dependency>-->

```

### What you expected to happen

Successfully run `MySqlLoadSqlParseTest`

### How to reproduce

Run `Successfully run MySqlLoadSqlParseTest`

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Sort

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/inlong-sort/sort-core/pom.xml
+++ b/inlong-sort/sort-core/pom.xml
@@ -203,6 +203,12 @@
                     <artifactId>sort-connector-oracle-cdc</artifactId>
                     <version>${project.version}</version>
                     <scope>test</scope>
+                    <exclusions>
+                        <exclusion>
+                            <groupId>io.debezium</groupId>
+                            <artifactId>debezium-ddl-parser</artifactId>
+                        </exclusion>
+                    </exclusions>
                 </dependency>
                 <dependency>
                     <groupId>org.apache.inlong</groupId>

